docs/modules/ROOT/pages/kotlin.adoc

```diff
@@ -82,8 +80,8 @@ declarations and expressive "`value or no value`" semantics without paying the c
 (Kotlin allows using functional constructs with nullable values. See this
 https://www.baeldung.com/kotlin-null-safety[comprehensive guide to Kotlin null-safety].)
 
-Although Java does not let one express null safety in its type-system, Reactor <<null-safety,now
-provides null safety>> of the whole Reactor API through tooling-friendly annotations declared
+Although Java does not let one express null safety in its type-system, Reactor xref:kotlin.adoc#kotlin-null-safety[now provides null safety]
```

Comment:
The link is incorrect - it should point to the section in advanced features / null-safety, not this section itself.

Reply:
fixed in https://github.com/reactor/reactor-core/pull/3806/commits/b80c9d3be77658c74bf28e3e5d1ab05e330394a4